### ISSUE TITLE
Color Grading

### ISSUE THREAD

**Issue by uuuussseer** [mifi/lossless-cut#807]
**Is your feature request related to a problem? Please describe.**
I usually use lossless-cut to quickly cut down videos into clips without losing video quality. When cutting down videos I usually find them to be a bit too gray, white, blue, red, etc..

**Describe the solution you'd like**
A way to color grade the videos and adjust things like the brightness, gamma, saturation, and hue.

**Describe alternatives you've considered**
Using a professional color grading program, but they are usually proprietary, complicated, cost a lot of money, and don't support Linux.

**Additional context**
Interesting Stack Overflow FFmpeg question that relates to this [here](https://stackoverflow.com/questions/58201025/ffmpeg-color-grading-video-material-and-display-original-source-as-picture-in#58211103). Also, thank you for making such a nice program that is open source! I (and many others) really appreciate it.

**Comment by mifi** [comment]
I actually did start working on a simple UI app for video color grading (using ffmpeg) with live wysiwyg preview, including LUT support. The problem was that it's quite slow. This functionality will probably never make it into losslesscut, as it is definitely not lossless. I added it to the list here: #372

I may finish the color grading app and make it public one day, but another reason why I stopped working on it is because i have an iPhone and apple released a really good color grading and video editing built into the photos app on iphone...
